PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl.rbxmx

```diff
@@ -68,11 +66,11 @@ function MasterControl:Init()
 			end
 			
 			local adjustedMoveValue = moveValue
-			if not NoVR and UserInputService.IsVREnabled then
-				local inverseHeadCFrame = UserInputService.UserHeadCFrame:inverse()
-				local yawOnlyLookVector = Vector3.new(inverseHeadCFrame.lookVector.X, 0, inverseHeadCFrame.lookVector.Z).unit
-				local yawOnlyCFrame = CFrame.new(Vector3.new(0,0,0), yawOnlyLookVector)
-				adjustedMoveValue = yawOnlyCFrame * adjustedMoveValue
+			if HasVRAPI and UserInputService.VREnabled and workspace.CurrentCamera.HeadLocked then
```

Comment:
HeadLocked is true by default and means that VR offset is being applied by the engine.